src/lib.rs

```diff
@@ -29,62 +40,68 @@ impl <const W: usize, const H: usize> Content<W, H> {
 /// This receives the `SmartLedsWriter` trait implementations along with a
 /// `MatrixType` that describes the size and the pixels mapping between the LED
 /// strip placement and the matrix's x y coordinates.
```

Comment:
This comment needs to be updated, because `MatrixType` no longer exists and the size is specified differently.